ios/FluentUI.Demo/FluentUI.Demo/Demos/PillButtonBarDemoController.swift

```diff
@@ -67,11 +78,12 @@ class PillButtonBarDemoController: DemoController {
         super.viewDidAppear(animated)
         if let window = view.window {
             filledBar?.backgroundColor = UIColor(light: Colors.primary(for: window), dark: Colors.Navigation.System.background)
+            customBar?.backgroundColor = UIColor(light: Colors.primary(for: window), dark: Colors.Navigation.System.background)
         }
     }
 
-    func createBar(items: [PillButtonBarItem], style: PillButtonStyle = .outline, centerAligned: Bool = false, disabledItems: Bool = false) -> UIView {
-        let bar = PillButtonBar(pillButtonStyle: style)
+    func createBar(items: [PillButtonBarItem], style: PillButtonStyle = .outline, centerAligned: Bool = false, disabledItems: Bool = false, useCustomPillsBackground: Bool = false) -> UIView {
+        let bar = useCustomPillsBackground ? PillButtonBar(pillButtonStyle: style, pillButtonBackgroundColor: Colors.Palette.successShade20.color) : PillButtonBar(pillButtonStyle: style)
```

Comment:
I think the ternary operator only on the parameter is more concise:

let bar = PillButtonBar(pillButtonStyle: style, pillButtonBackgroundColor: useCustomPillsBackground ? Colors.Palette.successShade20.color : nil)

Reply:
updated

---
In reply to: [468712817](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468712817) [](ancestors = 468712817)